File: Competetive programming/Python/Unit 2/cryptKicker.py
print("===== CRYPT KICKER =====\n")

encrypted = input("Enter encrypted text: ")

decrypted = ""

# Shift every letter back by 1
for ch in encrypted:

    if 'b' <= ch <= 'z':
        decrypted += chr(ord(ch) - 1)

    elif 'B' <= ch <= 'Z':
        decrypted += chr(ord(ch) - 1)

    elif ch == 'a':
        decrypted += 'z'

    elif ch == 'A':
        decrypted += 'Z'

    else:
        decrypted += ch

print("\nDecrypted Text:", decrypted)

File: Competetive programming/C/Unit 2/cryptKicker.c
#include <stdio.h>
#include <string.h>
int main() {
    char encrypted[100];
    char decrypted[100];
    printf("Enter encrypted text: ");
    fgets(encrypted, sizeof(encrypted), stdin);
    int i;

    for(i = 0; encrypted[i] != '\0'; i++) {

        if(encrypted[i] >= 'b' && encrypted[i] <= 'z')
            decrypted[i] = encrypted[i] - 1;

        else if(encrypted[i] >= 'B' && encrypted[i] <= 'Z')
            decrypted[i] = encrypted[i] - 1;

        else if(encrypted[i] == 'a')
            decrypted[i] = 'z';

        else if(encrypted[i] == 'A')
            decrypted[i] = 'Z';

        else
            decrypted[i] = encrypted[i];
    }

    decrypted[i] = '\0';

    printf("\nDecrypted Text: %s", decrypted);

    return 0;
}